cereal/log.capnp

```diff
@@ -868,6 +868,30 @@ struct ControlsState @0x97ff69c53601abf1 {
   vPidDEPRECATED @2 :Float32;
 }
 
+# Come up with a better name
+struct SmolModelDataV2 {
+  frameId @0 :UInt32;
+  frameIdExtra @1 :UInt32;
+
+  position @2 :SmolXYZTData;
+  orientation @3 :SmolXYZTData;
+  velocity @4 :SmolXYZTData;
+  orientationRate @5 :SmolXYZTData;
+
+  laneLines @6 :List(SmolXYZTData);
+  laneLineProbs @7 :List(Float32);
+  roadEdges @8 :List(SmolXYZTData);
```

Comment:
Do we need all of these things for metrics? I would propose something like this if it's possible:

struct DrivingModelData {
  frameId @0 :UInt32;
  frameIdExtra @1 :UInt32;
  action @2: ModelDataV2.Action;
  
  laneLineMeta @2: DrivingModelData.LaneLineMeta;

  struct LaneLineMeta {
    leftY
    rightY
    leftProb
    rightProb
}